File: arc/arc168/a.py
# https://atcoder.jp/contests/arc168/submissions/47771517

n = int(input())
s = input() + '<'

ans = 0
cnt = 0
for c in s:
  if c == '<':
    ans += cnt * (cnt + 1) // 2
    cnt = 0
  elif c == '>':
    cnt += 1

print(ans)

File: arc/arc168/a.cpp
// https://atcoder.jp/contests/arc168/submissions/47771536

#include <bits/stdc++.h>
using namespace std;
typedef long long ll;

int main() {
  ll n;
  cin >> n;

  string s;
  cin >> s;
  s += "<";

  ll ans = 0;
  ll cnt = 0;

  for(char c : s) {
    if(c == '<') {
      ans += cnt * (cnt + 1) / 2;
      cnt = 0;
    } else if(c == '>') {
      cnt++;
    }
  }

  cout << ans;
}
